website/docs/language/stacks/reference/tfdeploy.mdx

```diff
@@ -7,7 +7,7 @@ description: Stacks help you provision and coordinate your infrastructure lifecy
 
 A deployment configuration file defines how to deploy your Stack's infrastructure. Each Stack deployment runs in its agent, wholly isolated from other Stack deployments.
 
-Every Stack needs a deployment configuration file, `tfdeploy.hcl`, and this page describes all of the blocks you can use within a deployment configuration file. Note that none of the blocks in the deployment configuration file support [meta-arguments](/terraform/language/resources/syntax#meta-arguments).
+Every Stack needs a deployment configuration file, `tfdeploy.hcl`, and this page describes all of the blocks you can use within a deployment configuration file. Note that none of the blocks in the deployment configuration file support [meta-arguments](/terraform/language/meta-arguments).
```

Comment:
There is no `/terraform/language/meta-arguments` because there is no index page for that directory.

Reply:
I was going to name the meta-arguments concept page "index" and place it in that folder.